frontend/test/metabase-db/mongo/add.cy.spec.js

```diff
@@ -23,7 +23,7 @@ describe("mongodb > admin > add", () => {
     cy.contains("MongoDB").click({ force: true });
     cy.contains("Additional Mongo connection");
 
-    typeAndBlurUsingLabel("Name", "QA Mongo4");
+    typeAndBlurUsingLabel("Name", "QA Mongo4 ");
```

Comment:
Rather than adding whitespace to existing tests, it might be worth adding one test that's explicitly about this behavior. Two reasons:
- If I break this trimming behavior, a few tests will start failing. It would takes some debugging to see what the issue is causing those failures.
- As the tests evolve, we might drop the whitespace you added since it's not the main point of these tests. That would remove our coverage of this behavior.

I don't think we necessarily need to connect to a real db to test this. Asserting that the API call contains trimmed values seems like enough to me.

Reply:
Good point -- will add a test specifically for this.